### PR TITLE
add missing backsash character in the getting started guide

### PR DESCRIPTION
**Pull request by spring1843**
**1. Issue, if available:**

**2. Description of changes:**
- adds missing `\` to command
- avoid using a temp file
- add verification steps for two of the steps

**3. How was this change tested?**

**4. Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: *link to issue*
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/getting-started/_index.md [hunk 1]
```diff
@@ -82,8 +82,9 @@ You can add the details of the newly created EKS cluster to your local kubectl a
 ```bash
 aws eks update-kubeconfig --region ${AWS_DEFAULT_REGION} --name ${CLUSTER_NAME}
 ```
+Verify that you have a node with ready status by running `kubectl get nodes`.
```
Comment (suket22) [inline website/content/en/preview/getting-started/_index.md:85]: eksctl verifies that the nodes are `Ready` as part of creating the nodegroup so you don't need this.

File: website/content/en/preview/getting-started/_index.md [hunk 2]
```diff
@@ -82,8 +82,9 @@ You can add the details of the newly created EKS cluster to your local kubectl a
 ```bash
 aws eks update-kubeconfig --region ${AWS_DEFAULT_REGION} --name ${CLUSTER_NAME}
```
Comment (suket22) [inline website/content/en/preview/getting-started/_index.md:83]: I'm also not sure why this was added. eksctl will update your kubeconfig as part of creating the cluster

File: website/content/en/preview/getting-started/_index.md (hunk 1 above)
Comment (spring1843) [reply website/content/en/preview/getting-started/_index.md]: That makes sense, removed it.

File: website/content/en/preview/getting-started/_index.md (hunk 2 above)
Comment (spring1843) [reply website/content/en/preview/getting-started/_index.md]: removing

**Review by suket22 — APPROVED** [review]

**Pull request merged**